deploy/helm/flightctl/charts/ui/templates/flightctl-ui-cm.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   namespace: {{ .Release.Namespace }}
 data:
   FLIGHTCTL_SERVER: {{ .Values.api.url | quote }}
+  FLIGHTCTL_EXTERNAL_URL: '{{ if eq (include "flightctl.getServiceExposeMethod" .) "nodePort" }}https://api.{{ include "flightctl.getBaseDomain" . }}:{{ .Values.global.nodePorts.api }}/{{ else if and (eq (include "flightctl.getServiceExposeMethod" .) "gateway") (not (eq (int .Values.global.gatewayPorts.tls) 443)) }}https://api.{{ include "flightctl.getBaseDomain" . }}:{{ .Values.global.gatewayPorts.tls }}/{{ else }}https://api.{{ include "flightctl.getBaseDomain" . }}/{{ end }}'
```

Comment:
We already have a helper to get the API URL https://github.com/flightctl/flightctl/blob/main/deploy/helm/flightctl/templates/_helpers.tpl#L83 - lets use it.